website/docs/r/catalog_media.html.markdown

```diff
@@ -0,0 +1,39 @@
+---
+layout: "vcd"
+page_title: "vCloudDirector: vcd_catalog_media"
+sidebar_current: "docs-vcd-resource-catalog-media"
+description: |-
+  Provides a vCloud Director Media resource. This can be used to upload and delete media file in catalog.
+---
+
+# vcd\_catalog\_media
+
+Provides a vCloud Director Media resource. This can be used to upload media to catalog and delete it.
+
+## Example Usage
+
+```
+resource "vcd_catalog_media" "myNewMedia" {
+  org = "my-org"
+  catalog = "my-catalog" 
+
+  name = "my iso"
+  description = "new os versions 1.2"
```

Comment:
Nit: please remove the "1.2" from description to make eye-scanning text easier on the reader and to avoid unneeded thoughts what this version means.

Reply:
Updated